kolibri/plugins/facility/assets/src/routes.js

```diff
@@ -128,12 +132,20 @@ export default [
   },
   {
     path: '/:facility_id?/managesync',
+    props: route => {
+      return { facilityId: route.params.facilityId || store.getters.currentFacilityId };
+    },
     component: ManageSyncSchedule,
-    name: PageNames.MANAGE_SYNC_SCHEDULE,
+    name: SyncPageNames.MANAGE_SYNC_SCHEDULE,
   },
   {
-    path: '/:facility_id?/editdevice/:deviceId',
+    path: '/:facility_id?/editdevice/:deviceId/',
     component: editDeviceSyncSchedule,
-    name: PageNames.EDIT_SYNC_SCHEDULE,
+    name: SyncPageNames.EDIT_SYNC_SCHEDULE,
+    props: route => {
+      return { facilityId: route.params.facilityId || store.getters.currentFacilityId };
```

Comment:
This should be `route.params.facility_id` - the name of the parameter matches what is specified in the route path:
`path: '/:facility_id?/editdevice/:deviceId/',`